shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -662,6 +665,9 @@ int calculateBottomKeyboardInset(WindowInsets insets) {
     // the wider than expected padding when the status and navigation bars are hidden.
     @Override
     public final WindowInsets onApplyWindowInsets(WindowInsets insets) {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.KITKAT_WATCH) {
```

Comment:
And a comment here, too.

Reply:
The Javadoc on this method also mentions that this isn't called on API < 20. Do you think that's sufficient, or do you want more detail there saying that if it is deliberately called on those SDKs it'll return null as a non-op?